packages/ui-app/src/InputStorage/index.tsx

```diff
@@ -19,7 +19,7 @@ import classes from '../util/classes';
 import translate from '../translate';
 import SelectKey from './SelectKey';
 import SelectSection from './SelectSection';
-import keyOptions from './options/key';
+import methodOptions from './options/method';
```

Comment:
Not sure why this is done - in storage the term "method" doesn't make much sense since it actually refers to a key in the state storage. (For extrinsics methods are fine since that is what they are)